.github/workflows/builder_image_tests.yml

```diff
@@ -25,12 +26,16 @@ jobs:
       matrix:
         os: [ ubuntu-20.04 ]
         include:
+          - quarkus-version: '2.7.7.Final'
+            mandrel-builder-image: '21.3-java17'
```

Comment:
Those two lines can probably go. It's EOL.

Reply:
as long as you can pick it here... https://code.quarkus.redhat.com/  (top dropdown)